src/filter_parameters.rs

```diff
@@ -19,8 +25,85 @@ pub struct FilterParameters {
     pub drive: ParameterF32,
     pub mode: ParameterUsize,
     pub slope: ParameterUsize,
-    pub filter_type: AtomicUsize,
+    pub filter_type: ParameterUsize,
+
+    pub reserved: ParameterUsize,
+}
+
+#[repr(i32)]
+#[derive(FromPrimitive, Eq, PartialEq, Debug, EnumIter)]
+pub enum FilterParameterNr {
+    Cutoff,
+    Res,
+    Drive,
+
+    FilterType = 32,
+    Mode,
+    Slope,
+
+    #[num_enum(default)]
+    Padding = 200,
```

Comment:
Is this for padding the preset files? If so, very cool

Reply:
Since it's just a filter plugin, maybe change to a lower, more reasonable value. Even if I started adding LFOs and more filters it'd probably not go beyond 50